src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ToolTipTests/ToolTip_Tests.cs

```diff
@@ -120,5 +120,31 @@ public void ToolTip_WhenHostCollapsed_Test()
 			// its tooltip should disappear as well.
 			tooltipText.ShouldNotBeVisible();
 		}
+
+		[Test]
+		[AutoRetry]
+		public void ToolTip_DataContext()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.ToolTip.Tooltip_Content_Binding");
+			var dataContextValueTextBlock = _app.Marked("DataContextValueTextBlock");
+			var setDataContextValueTextBlockButton = _app.Marked("SetDataContextValueTextBlockButton");
+			var tooltip = _app.Marked("MyToolTip");
+			var updateDataContextButton = _app.Marked("UpdateDataContextButton");
+
+			Assert.AreEqual("Initial text", dataContextValueTextBlock.GetDependencyPropertyValue<string>("Text"));
+
+			setDataContextValueTextBlockButton.FastTap();
+			Assert.AreEqual("1", dataContextValueTextBlock.GetDependencyPropertyValue<string>("Text"));
+			setDataContextValueTextBlockButton.FastTap();
+			Assert.AreEqual("1", dataContextValueTextBlock.GetDependencyPropertyValue<string>("Text"));
+
+			updateDataContextButton.FastTap();
+			setDataContextValueTextBlockButton.FastTap();
+			// TODO: This is a bug. It should be 2
+			Assert.AreEqual("1", dataContextValueTextBlock.GetDependencyPropertyValue<string>("Text"));
+			setDataContextValueTextBlockButton.FastTap();
+			// TODO: This is a bug. It should be 2
+			Assert.AreEqual("1", dataContextValueTextBlock.GetDependencyPropertyValue<string>("Text"));
```

Comment:
I spent too much trying to get this scenario to work. But I always break something.

Reply:
I'll open a new issue for this if the PR got merged.

Reply:
It's probably best if create a runtime test for that instead (and it'll run on all platforms). You can use internal APIs that tap a button and then test using `FindName`.